object-construction-checker/tests/socket/ACSocketTest.java

```diff
@@ -479,12 +477,28 @@ private SSLSocket configureSSLSocket(@Owning SSLSocket socket, boolean isClientS
 
 
     private void updateSocketAddresses( SelectionKey sockKey ) {
-        // error: required.method.not.called
```

Comment:
Just so we don't forget, after chatting this morning @Nargeshdb and I think there should still be a warning here, so we have a bug somewhere

Reply:
https://github.com/kelloggm/object-construction-checker/pull/242/commits/0aff468930f00936d5c80c5f307eddc1fff93edb